build/configs/default_config.yaml

```diff
@@ -97,7 +97,7 @@ queue:
 
 # CentChain specific configuration
 centChain:
-  nodeURL: ws://127.0.0.1:9944
+  nodeURL: ws://127.0.0.1:9946
```

Comment:
Does the fact that we changed the ports in our local env to be `para_id` + `9946` matter here?

Reply:
The chain service is meant to be run here as docker and we do not support different ports.

Reply:
Great!